core/common/modules/ajax/module.php

```diff
@@ -266,17 +266,7 @@ private function send_success() {
 			ob_end_clean();
 		}
 
-		if ( function_exists( 'gzencode' ) ) {
-			$response = gzencode( $json );
-
-			header( 'Content-Type: application/json; charset=utf-8' );
-			header( 'Content-Encoding: gzip' );
-			header( 'Content-Length: ' . strlen( $response ) );
-
-			echo $response; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-		} else {
-			echo $json; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-		}
+		echo $json; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
```suggestion
		header( 'Content-Type: application/json; charset=UTF-8' );
		echo $json; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```
@Dekadinious Can you please apply the suggestion?